pkg/validations/createvalidations/preflightvalidations.go

```diff
@@ -18,8 +18,13 @@ func (u *CreateValidations) PreflightValidations(ctx context.Context) (err error
 	createValidations := []validations.ValidationResult{
 		{
 			Name:        "validate taints support",
-			Remediation: "",
-			Err:         ValidateTaintsSupport(ctx, u.Opts.Spec),
+			Remediation: "ensure TAINTS_SUPPORT env variable is set",
+			Err:         ValidateTaintsSupport(u.Opts.Spec),
+		},
+		{
+			Name:        "validate taints support",
+			Remediation: "ensure NODE_LABELS_SUPPORT env variable is set",
+			Err:         ValidateNodeLabelsSupport(u.Opts.Spec),
```

Comment:
nit: I think we should put feature flag validations somewhere else. Putting them here prints a message with the validation name even when it passes (I think). And I believe feature flag checks should "silent"